internal/envoy/bootstrap.go

```diff
@@ -204,6 +204,10 @@ type BootstrapConfig struct {
 	// Defaults to 127.0.0.1.
 	XDSAddress string
 
+	// XDSClusterName is the name reference for the gROC XDS management server.
+	// Defaults to contour.
+	XDSClusterName string
```

Comment:
Shall it remove this part as well and go back to `ConfigSource("contour")` (e.g. remove the carrier variable).